### PR TITLE
Now it's possible to star contacts

### PR DESCRIPTION
**Pull request by Julian1998**
Also I fixed a little tooltip error when hovering about the delte icon

### CODE REVIEW

**Comment by Julian1998** [comment]
@DeepDiver1975 Can you explain me why codecov is failing?

File: js/components/contactDetails/contactDetails_controller.js [hunk 1]
```diff
@@ -23,14 +23,26 @@ angular.module('contactsApp')
 		placeholderTitle : t('contacts', 'Title'),
 		selectField : t('contacts', 'Add field ...'),
 		download : t('contacts', 'Download'),
-		delete : t('contacts', 'Delete')
+		delete : t('contacts', 'Delete'),
+		favour : t('contacts', 'Favour'),
```
Comment (DeepDiver1975) [inline js/components/contactDetails/contactDetails_controller.js:27]: -u

File: templates/contactDetails.html [hunk 2]
```diff
@@ -24,8 +24,10 @@ <h2>
 				</div>
 			</div>
 			<div id="details-actions">
+				<button ng-show="!ctrl.starred" ng-click="ctrl.starContact()" class="icon-star" title="{{ctrl.t.favour}}"></button>
```
Comment (DeepDiver1975) [inline templates/contactDetails.html:27]: two buttons? why not change the class inside.

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

File: templates/contactDetails.html (hunk 2 above)
Comment (Julian1998) [reply templates/contactDetails.html:27]: Didn't thought about that! Good point

File: js/components/contactDetails/contactDetails_controller.js (hunk 1 above)
Comment (Julian1998) [reply js/components/contactDetails/contactDetails_controller.js]: -u? What do you mean by that

**Comment by DeepDiver1975** [comment]
:coffin:

**Pull request closed without merging**